<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="22720" started-at="2020-07-22T18:11:45 EDT" finished-at="2020-07-22T18:12:08 EDT">
    <groups>
    </groups>
    <test name="Default test" duration-ms="22720" started-at="2020-07-22T18:11:45 EDT" finished-at="2020-07-22T18:12:08 EDT">
      <class name="FirstMavenTNG.testproj.TestNGProj">
        <test-method status="PASS" signature="setExtent()[pri:0, instance:FirstMavenTNG.testproj.TestNGProj@53f65459]" name="setExtent" is-config="true" duration-ms="98" started-at="2020-07-22T18:11:45 EDT" finished-at="2020-07-22T18:11:45 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setExtent -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:FirstMavenTNG.testproj.TestNGProj@53f65459]" name="setUp" is-config="true" duration-ms="7130" started-at="2020-07-22T18:11:46 EDT" finished-at="2020-07-22T18:11:53 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="loginValidTest(java.lang.String, java.lang.String)[pri:0, instance:FirstMavenTNG.testproj.TestNGProj@53f65459]" name="loginValidTest" duration-ms="4006" started-at="2020-07-22T18:11:53 EDT" data-provider="getLoginData" finished-at="2020-07-22T18:11:57 EDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[mngr272486]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[qUmYrus]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginValidTest -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:FirstMavenTNG.testproj.TestNGProj@53f65459]" name="tearDown" is-config="true" duration-ms="692" started-at="2020-07-22T18:11:57 EDT" finished-at="2020-07-22T18:11:58 EDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=loginValidTest status=SUCCESS method=TestNGProj.loginValidTest(java.lang.String, java.lang.String)[pri:0, instance:FirstMavenTNG.testproj.TestNGProj@53f65459] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:FirstMavenTNG.testproj.TestNGProj@53f65459]" name="setUp" is-config="true" duration-ms="6369" started-at="2020-07-22T18:11:58 EDT" finished-at="2020-07-22T18:12:04 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="loginValidTest(java.lang.String, java.lang.String)[pri:0, instance:FirstMavenTNG.testproj.TestNGProj@53f65459]" name="loginValidTest" duration-ms="3965" started-at="2020-07-22T18:12:04 EDT" data-provider="getLoginData" finished-at="2020-07-22T18:12:08 EDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[mngr272486]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[er3442ff]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [User or Password is not valid12] but found [User or Password is not valid]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [User or Password is not valid12] but found [User or Password is not valid]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at FirstMavenTNG.testproj.TestNGProj.loginValidTest(TestNGProj.java:99)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginValidTest -->
        <test-method status="FAIL" signature="tearDown(org.testng.ITestResult)[pri:0, instance:FirstMavenTNG.testproj.TestNGProj@53f65459]" name="tearDown" is-config="true" duration-ms="8" started-at="2020-07-22T18:12:08 EDT" finished-at="2020-07-22T18:12:08 EDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=loginValidTest status=FAILURE method=TestNGProj.loginValidTest(java.lang.String, java.lang.String)[pri:0, instance:FirstMavenTNG.testproj.TestNGProj@53f65459] output={null}]]]>
              </value>
            </param>
          </params>
          <exception class="org.openqa.selenium.UnhandledAlertException">
            <message>
              <![CDATA[unexpected alert open: {Alert text : User or Password is not valid}
  (Session info: chrome=83.0.4103.116): User or Password is not valid
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:48'
System info: host: 'DESKTOP-NDEI415', ip: '192.168.1.84', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_251'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 83.0.4103.116, chrome: {chromedriverVersion: 83.0.4103.39 (ccbf011cb2d2b..., userDataDir: C:\Users\kandh\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:57788}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:virtualAuthenticators: true}
Session ID: 18357d2bb96690a62baa398b748fafc1]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.UnhandledAlertException: unexpected alert open: {Alert text : User or Password is not valid}
  (Session info: chrome=83.0.4103.116): User or Password is not valid
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:48'
System info: host: 'DESKTOP-NDEI415', ip: '192.168.1.84', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_251'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 83.0.4103.116, chrome: {chromedriverVersion: 83.0.4103.39 (ccbf011cb2d2b..., userDataDir: C:\Users\kandh\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:57788}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:virtualAuthenticators: true}
Session ID: 18357d2bb96690a62baa398b748fafc1
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:120)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:609)
at org.openqa.selenium.remote.RemoteWebDriver.getScreenshotAs(RemoteWebDriver.java:295)
at FirstMavenTNG.testproj.TestNGProj.getScreenshot(TestNGProj.java:53)
at FirstMavenTNG.testproj.TestNGProj.tearDown(TestNGProj.java:129)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:64)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:364)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:318)
at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:700)
at org.testng.internal.TestInvoker.runAfterGroupsConfigurations(TestInvoker.java:676)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:660)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.UnhandledAlertException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="SKIP" signature="endReport()[pri:0, instance:FirstMavenTNG.testproj.TestNGProj@53f65459]" name="endReport" is-config="true" duration-ms="-1595455928471" started-at="2020-07-22T18:12:08 EDT" finished-at="1969-12-31T19:00:00 EST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endReport -->
      </class> <!-- FirstMavenTNG.testproj.TestNGProj -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
